app/experimenter/experiments/constants/nimbus.py

```diff
@@ -409,6 +409,30 @@ class NimbusTargetingConfig:
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+TARGETING_INFREQUENT_USER_NEED_DEFAULT_HAS_PIN = NimbusTargetingConfig(
+    name="Infrequent user (need default, has pin)",
+    slug="infrequent_user_need_default_has_pin",
+    description="Between 1 & 6 days activity in past 28 days need default w/ pin",
+    targeting=("{infrequent_need_default} && !doesAppNeedPin || {sticky}").format(
+        infrequent_need_default=TARGETING_INFREQUENT_USER_NEED_DEFAULT.targeting,
+        sticky=TARGETING_STICKY,
+    ),
+    desktop_telemetry="",
+    application_choice_names=(Application.DESKTOP.name,),
+)
+
+TARGETING_INFREQUENT_USER_HAS_DEFAULT_NEED_PIN = NimbusTargetingConfig(
+    name="Infrequent user (has default, need pin)",
+    slug="infrequent_user_need_pin_has_default",
```

Comment:
Change the slug word ordering to match the name & desc word ordering here and elsewhere, please.